examples/protocol_exp/PromiseTensor and Protocol Roadmap.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>I think making the method <code>move</code>  do something other than moving the tensor is going to be a surprising and non-intuitive API design choice. Can we keep the existing semantics of <code>move</code>  and add a second method that extends the graph across workers?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2959/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
ack(acknowledged!) @iamtrask ^ your thoughts?

Reply:
Sounds good to me (and should be an easy change).

This does beg the question though, what does .move() do in this context? I suppose it would move the PromiseTensor without extending the graph?